module1.js

```diff
@@ -111,6 +136,10 @@ function question7(array1, array2, extraElement) {
 
 function question8(retailPrice, taxAmount) {
   // TODO:
+  var ret = retailPrice.toCurrency();
+  var salesTax = ret * taxAmount;
+  
+  return salesTax;
```

Comment:
In general there is no point assigning a variable only to return it in the following line; just `return ret * taxAmount;`.